.changeset/curly-lizards-clap.md

```diff
@@ -0,0 +1,6 @@
+---
+"@lens-protocol/api-bindings": minor
+"@lens-protocol/react": minor
+---
+
+Added 'profilePublicationsForSale' hook
```

Comment:
I understand it is best to mark only the `react` package in the changelog like this, easier to build the final changelog for future release.